### ISSUE TITLE
Suggestion: Add Carthage compatibility badge to README

### ISSUE THREAD

**Issue by gbrixey** [Iterable/swift-sdk#60]
[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```
[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```

**Comment by tapashmajumder** [comment]
We will looking into this soon.

**Comment by vbabenkoru** [comment]
Resolved via #105.
